crates/librqbit_core/src/torrent_metainfo.rs

```diff
@@ -37,7 +43,10 @@ pub struct TorrentMetaV1<BufType> {
         skip_serializing_if = "Vec::is_empty"
     )]
     pub announce_list: Vec<Vec<BufType>>,
-    pub info: TorrentMetaV1Info<BufType>,
+    /// Unmodified bytes corresponding to the info value. Used for computing infohash and
+    /// communicated to peers via the metadata extension.
+    #[serde(borrow = "'a")]
+    pub info: Option<RawInfo<'a, BufType>>,
```

Comment:
@ikatson I don't suppose you could check this branch out in your IDE and help me solve the single compile error I have left? I am trying to support deserializing from borrowed data, but I get a lifetime I just can't figure out for having the raw info bytes in this field. It the lifetime should be the deserializer lifetime, but I don't know how to express it.